docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -0,0 +1,296 @@
+# Provisioning hosted clusters on MCE (Without Hypershift Deployment)
+
+Configuring hosted control planes requires a hosting service cluster and a hosted cluster. By deploying the HyperShift operator on an existing cluster, you can make that cluster into a hosting service cluster and start the creation of the hosted cluster. 
+
+Hosted control planes is a Technology Preview feature, so the related components are disabled by default. Enable the feature by editing the `multiclusterengine` custom resource to set the `spec.overrides.components[?(@.name=='hypershift-preview')].enabled` to `true`. 
```

Comment:
I think it will be easier to read and understand if we use an example like the below instead of this json path string

```
  spec:
    overrides:
      components:
      - enabled: true
        name: hypershift-preview
...
```

Reply:
Kept it this way to be consistent with the current iteration of the [official doc](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#hosted-control-planes-intro), and I'm guessing we'll be able to remove this part entirely when [we turn hypershift on by default](https://issues.redhat.com/browse/ACM-1659).